config/m4/ucm.m4

```diff
@@ -90,9 +90,11 @@ AS_IF([test "x$mmap_hooks_happy" == "xyes"],
       AS_IF([test "x$ipc_hooks_happy" == "xyes" -o "x$shm_hooks_happy" == "xyes"],
             [bistro_hooks_happy=yes]))
 
-AS_IF([test "x$bistro_hooks_happy" == "xyes"],
+AS_IF([test "x$bistro_hooks_happy" == "xyes__"],
```

Comment:
yes__ ?

Reply:
right__
tested all config combinations - forgot to clean last experiment